canvas_modules/common-canvas/src/common-properties/components/properties-buttons/properties-buttons.jsx

```diff
@@ -54,7 +54,7 @@ class PropertiesButtons extends Component {
 					data-id="properties-apply-button"
 					className="properties-apply-button"
 					type="button"
-					size="small"
+					size="xl"
```

Comment:
How much does this change the button size?

Reply:
It doesn't. In [test harness](https://elyra-canvas-test-harness.u20youmx4sm.us-south.codeengine.appdomain.cloud/#/), height of properties buttons is `64px`. It was adding `height: 4rem` CSS which isn't needed anymore because `xl` buttons have height 64px.
![Screenshot 2024-03-22 at 10 51 13 AM](https://github.com/elyra-ai/canvas/assets/25124000/49a1966d-b641-409a-8004-39b5e5b3b8e0)


Height of `xl` buttons is also `64px` in localhost -
![Screenshot 2024-03-22 at 10 51 25 AM](https://github.com/elyra-ai/canvas/assets/25124000/9f37604a-c6c7-473f-a44f-1269c5e6e026)